blog/2022-1-2-your-first-extension.md

```diff
@@ -0,0 +1,40 @@
+---
+slug: your-first-extension
+title: Your first extension
+authors: [CharlieS1103]
+tags: [spicetify, documentation, community,development]
+---
+
+## So you want to make an extension....
+
+
+### First step:
+Start out by setting up your environment: 
+In terminal or powershell, run ``spicetify enable-devtool apply``
```

Comment:
I believe you do not need the `apply`.

Reply:
@afonsojramos I kind of prefer the apply being there as it will reload Spotify with the changes immediately, which isn't really important as they'll run it in the future regardless, but also why not I guess.

Reply:
I get you, but on the `enable-devtool` command Spotify does not need to be reloaded at all.